src/components/HTMLEngineProvider/htmlEngineUtils.ts

```diff
@@ -7,12 +8,12 @@ const MAX_IMG_DIMENSIONS = 512;
  * is used by the HTML component in the default renderer for img tags to scale
  * down images that would otherwise overflow horizontally.
  *
- * @param {string} tagName - The name of the tag for which max width should be constrained.
- * @param {number} contentWidth - The content width provided to the HTML
+ * @param tagName - The name of the tag for which max width should be constrained.
+ * @param contentWidth - The content width provided to the HTML
  * component.
- * @returns {number} The minimum between contentWidth and MAX_IMG_DIMENSIONS
+ * @returns The minimum between contentWidth and MAX_IMG_DIMENSIONS
  */
-function computeEmbeddedMaxWidth(tagName, contentWidth) {
+function computeEmbeddedMaxWidth(contentWidth: number, tagName: string): number {
```

Comment:
This is interesting were we doing something wrong the whole time? Isn't this param swapping a breaking change?